### PR TITLE
Sepand/refactor browse

### PR DESCRIPTION
**Pull request by DSep**
- Made further changes to file structure. 
- Fixed redux bug. 
- Added numerous constant files.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 CoursefFlo (subject to name change) is a webapp designed to help students see the required pre-req and co-req’s that they need before graduation in a visually pleasing manner, and allows users to explore their degree programs to better understand their options. It will store data in the form of user login, preferences, saved courses, and recent searches; we will be scraping and storing all the SSC course data, every time it is updated by UBC. Our additional functionalities that we could add or remove are: a complete API system to scrape and parse SSC data as a standalone repository, a visualizer that can display courses in 3d, and a recommender system that can give suggestions towards where we can take further courses in the future.
 
-Team: [Sepand](https://github.com/DSep), [Sam](https://github.com/sam-ip), [Brandon](https://github.com/beetai), [Guanting](https://github.com/baconandchip).
+Team: [Sepand](https://github.com/DSep), [Sam](https://github.com/sam-ip), [Brandon](https://github.com/beetai), [Guanting](https://github.com/baconandchips).
```
Comment (baconandchips) [inline README.md:3]: me like

**Review by yqj2k — APPROVED** [review]
LGTM. I like the refactoring you have made. Again, open to having more refactoring down the road as we add more technologies and other ideas as we see fit.

File: src/components/Browse.tsx [hunk 2]
```diff
@@ -126,9 +113,8 @@ const Browse = (props: any) => {
     if (!Array.isArray(newValue) || newValue.length !== 2) {
       return;
     }
-    console.log("old val", searchInputs.numberRange);
-    console.log("new val", newValue);
-    searchInputs.numberRange = newValue;
+    const newRange: number | number[] = newValue;
```
Comment (baconandchips) [inline src/components/Browse.tsx]: Under what instances does newRange offer a number? Other than this part that I don't understand, I think everything else looks good. (If you don't mind, I'll be viewing replies after 7am tomorrow.)

Comment (DSep) [reply src/components/Browse.tsx]: I was wondering this myself, but I dont actually know. I was following the material UI guide, in TypeScript, and their event seemed to have the possibility of coming as either type, so I left it as such. 

However, look at line 120 and you see I've actually handled that, validating for arrays with 2 items.

**Review by beetai — COMMENTED** [review]
Isn't the naming convention that actions are lower case (i.e. `browse.tsx`)? I noticed you renamed it to `Browse.tsx`.
Apart from this, LGTM.

File: src/components/Browse.tsx (hunk 2 above)
Comment (baconandchips) [reply src/components/Browse.tsx]: Ok sounds good! LGTM as well

**Review by baconandchips — APPROVED** [review]

**Pull request merged**